omod/src/main/webapp/fragments/patientdashboard/visits.gsp

```diff
@@ -37,6 +49,10 @@
 <% } %>
 <!-- End of encounter templates -->
 
+<i id="i-toggle" title="${ui.message('coreapps.expandAll')}"
+   class="toggle-icon icon-arrow-down small caret-color"
```

Comment:
I don't know if a down arrow is right for this. Since each encounter just has the text "show details" shouldn't we do the equivalent here? e.g. "show all encounter details"